### DISCUSSION TITLE
🎉 New Community Manager

### DISCUSSION THREAD

**Discussion by Rumyra** [mdn/mdn-community#560] [Content]
Hey everyone! I wanted to announce we have a new Community Manager \o/ @schalkneethling 

Schalk has been an engineer working on MDN for over four years and has recently transitioned into the Community Manager role. He brings a huge amount of experience and knowledge which will, I'm sure, benefit the whole community. He's already started implementing some great improvements (both here on GitHub and across other channels).

Check out the new [MDN-Community repository](https://github.com/mdn/mdn-community/discussions) which Schalk has created as a hub for community discussions, questions, announcements and more!

… and expect more updates about the work we are doing to improve how we work as a community together.

**Comment by hamishwillee** [comment]
Congratulations @schalkneethling - thank you for agreeing to herd the cats" :-).

Nice to see you getting the title for the role you've been kind of doing for ages!

**Reply by schalkneethling** [reply]
Thank you so much, @hamishwillee 👍

**Reply by Abhiram-Gullapalli** [reply]
Congratulations @schalkneethling 🎉 .

**Reply by schalkneethling** [reply]
Thank you, @gullapalli9482
